services/logs/logs.go

```diff
@@ -1,5 +1,5 @@
 // pmm-managed
-// Copyright (C) 2017 Percona LLC
+// Copyright (C) 2018 Percona LLC
```

Comment:
Nope. Copyright year should be updated only for "significant" changes. And linter will not allow you to do that.